CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Change history for ui-plugin-find-po-line
 
 ## (4.1.0 IN PROGRESS)
+* Optimize query when selecting a fund for po lines. Refs UIOR-1100.
```

Comment:
you can't merge to this repo UIOR, move ticket in jira to right place and update changelog, PR name etc